doc/advanced-configuration/Listener-modules.md

```diff
@@ -16,14 +16,15 @@ You only need to declare running `ejabberd_c2s`, to have the other 2 modules sta
 
 ### Configuration
 
-* `certfile` (string, default: no certfile will be used) - Path to the X509 PEM file with a certificate and a private key (not protected by a password).
+* `certfile` (string, default: no certfile will be used) - Path to the X509 PEM file with a certificate and a private key (not protected by a password). If the certificate is signed by an intermediate CA, you should specify here the whole CA chain.
 * `starttls` (default: disabled) - Enables StartTLS support; requires `certfile`.
 * `starttls_required` (default: disabled) - enforces StartTLS usage.
+* `tls` (default: disabled) - enables the old SSL connection.
```

Comment:
What does 'old' mean? (I know it but the reader might not :))

Reply:
you're right, its not informative - I just took it from ejabberd.cfg file [1]
Can be changed in both files then

[1] https://github.com/esl/MongooseIM/blob/master/rel/files/ejabberd.cfg#L238